kernels/set_column.cl

```diff
@@ -0,0 +1,14 @@
+__constant sampler_t sampler = CLK_NORMALIZED_COORDS_FALSE | CLK_ADDRESS_CLAMP_TO_EDGE | CLK_FILTER_NEAREST;
+
+__kernel void set_column(
+    IMAGE_dst_TYPE  dst,
+    const int       cst,
+    const float     scalar
+)
+{
+  const int x = cst;
```

Comment:
```suggestion
    const int       column,
    const float     scalar
)
{
  const int x = column;
```

Reply:
`cst` stand for constant, we can use something else. I would suggest `index`, avoiding the trinity `column`, `slice`, `row` tags.